api/services/ocean/aws/paths/cluster-roll.yaml

```diff
@@ -66,7 +66,14 @@ get:
       example: o-1abcd124
       description: >
         The ID of the Ocean
-    - $ref: "../../commons/parameters/clusterRollId.yaml"
```

Comment:
@vlad666spot please check as it seems to be a repeated issue - 
maybe we should change 'query' to 'path' for all  "../../commons/parameters/clusterRollId.yaml" usages,
so no need to do it multiple times

Reply:
@sagishoshan fixed